code/__DEFINES/supermatter.dm

```diff
@@ -0,0 +1,61 @@
+#define SUPERMATTER_MAXIMUM_ENERGY 1e6
+
+#define PLASMA_HEAT_PENALTY 15     // Higher == Bigger heat and waste penalty from having the crystal surrounded by this gas. Negative numbers reduce penalty.
+#define OXYGEN_HEAT_PENALTY 1
```

Comment:
Moving these defines over to a seperate file seems counterproductive to me.

Reply:
They need to be global because they're used elsewhere. Leaving them in the SM file means they're not global.